lib/internal_metric/global.rb

```diff
@@ -359,8 +359,8 @@ def find_stuck_sidekiq_jobs
       stats = {}
       Sidekiq::Workers.new.each do |queue, tid, work|
         next unless queue.start_with?(hostname)
-        next if Time.at(work["run_at"]) >= (Time.now - 60 * STUCK_SIDEKIQ_JOB_MINUTES)
-        labels = { job_name: work.dig("payload", "class") }
+        next if Time.at(work.run_at) >= (Time.now - 60 * STUCK_SIDEKIQ_JOB_MINUTES)
+        labels = { job_name: JSON.parse(work.payload)["class"] }
```

Comment:
Sidekiq already parses the payload, the job and its class is available at `work.job.item["class"]`.